content/work/publicfinance/opencontractingindiaassam/index.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Open Contracting India - Assam
+image: ./image.png
+summary: Fixing Public Procurement with Public data!
+context: 
+solution: We will explore the following 4 priority areas and following a rigorous assessment, identify the top two options that are most likely to translate into meaningful reforms that will deliver life changing impacts for the people of Assam - Child Health, Floods Management, COVID-19 Relief Efforts, Water & Sanitation. Develop a basic replicable open data and analytical platform - Public Procurement Suite. We hope tp co create a  suite that will enable users to download raw data as well as data mapped to Open Contracting Data Standards in machine-readable format moving away from scanned images/pdfs/CAPTCHAs.
+aim:
+url:
+twitter:
+linkedin:
+github:
+newsletter:
+sector: Public Finance
+type: project
+---
+
+How the government spends money on public goods, services, and infrastructure is referred to as Public Procurement. Public procurement data is imperative to understand government priorities and advocate for better spending. 
+We are currently in brainstorming phase for following two projects
+
+LIFT - We are part of 7 teams across the globe working for public procurement reforms across different geographies over the span of next 18 months via OpenData & participatory processes. We look forward to learn more from  We want to use public procurement to improve equitable access to flood relief and control of goods, services and works in Assam. We define our ultimate success as substantial progress in the next 3 years in realizing SDG 11.5. The measurable impact we want to work toward is:  Equitable distribution of public goods, services and works procured for flood relief such as temporary shelters, relief packages, restoration of damaged areas and flood management resources such as construction of flood barriers, soil erosion prevention and soil recovery services so that public spending matches citizens’ needs.

```

Comment:
```suggestion
LIFT - We are part of 7 teams across the globe working for public procurement reforms across different geographies over the span of the next 18 months via OpenData & participatory processes. We look forward to learning more from  We want to use public procurement to improve equitable access to flood relief and control of goods, services, and works in Assam. We define our ultimate success as substantial progress in the next 3 years in realizing SDG 11.5. The measurable impact we want to work toward is Equitable distribution of public goods, services, and works procured for flood relief such as temporary shelters, relief packages, restoration of damaged areas, and flood management resources such as the construction of flood barriers, soil erosion prevention, and soil recovery services so that public spending matches citizens’ needs.

```

Reply:
Missing sentence - "We look forward to learn more from..."

Reply:
  Cool